frontend/src/components/PathAppLayout.tsx

```diff
@@ -0,0 +1,110 @@
+        passing the <span className={styles.red}>IDEX California® Examination</span>, these
+        candidates must submit the required proof of education/work experience, as listed under the
+        Categories of Certification, in order to complete the certification process.
+      </p>
+    ),
+
+    2: (
+      <p>
+        Path 2 is for Applicants who meet minimum education and education/work experience, may take
+        the <span className={styles.red}>IDEX California® Examination</span>. After successfully
+        passing the <span className={styles.red}>IDEX California® Examination</span>, these
+        candidates must submit the required proof of education/work experience, as listed under the
+        Categories of Certification, in order to complete the certification process.
+      </p>
+    ),
+
+    3: (
+      <p>
+        Path 3 is for Applicants who meet minimum education and education/work experience, and has
+        passed one of the qualifying National Interior Designer Exams. Applicants are eligible and
+        must take the <span className={styles.red}>IDEX California® Examination</span>. After
+        successfully passing the <span className={styles.red}>IDEX California® Examination</span>,
+        these candidates must submit the required proof of education/work experience, as listed
+        under the Categories of Certification, in order to complete the certification process.
+      </p>
+    ),
+
+    4: (
+      <p>
+        Path 4 is for Applicants who meet minimum education and education/work experience, have
+        passed one of the qualifying National Interior Designer Exams, and passed required ICC
+        courses. Applicants are eligible and must take the{" "}
+        <span className={styles.red}>IDEX California® Examination</span>. After successfully
+        passing the <span className={styles.red}>IDEX California® Examination</span>, these
+        candidates must submit the required proof of education/work experience, as listed under the
+        Categories of Certification, in order to complete the certification process.
+      </p>
+    ),
+  };
+
+  return (
+    <Page>
+      <div className={styles.pathwayApplicationBase}>
+        <div className={styles.applicationContainer}>
+          <h1 className={styles.title}>Path {path} Application</h1>
+
+          <div className={styles.pathContent}>{path_descriptions[path]}</div>
+          <div className={styles.centeredContainer}>
+            <Button onClick={null}>Retake Prescreening Questions</Button>
+          </div>
+
+          <div className={styles.note}>
+            <p className={styles.red}>
+              {" "}
+              If you would like to be sorted into a different pathway, please click above to retake
+              the
+              <br></br>
+              sorting questionnaire. For further inquiries, please reach out to{" "}
+              <strong>ccidc@ccidc.org</strong>
```

Comment:
can we also make this a link using the mailto property? It seems resonable  that when they click the email it just opens up the email app of the use. It can be done this way

`<a href="mailto:name@example.com">Send an email!</a>`